bc_obps/compliance/models/compliance_penalty.py

```diff
@@ -14,6 +14,13 @@ class Frequency(models.TextChoices):
         DAILY = 'Daily'
         MONTHLY = 'Monthly'
 
+    class Status(models.TextChoices):
+        NONE = 'NONE', 'None'
+        ACCRUING = 'ACCRUING', 'Accruing'
+        DUE = 'DUE', 'Due'
+        NOT_PAID = 'NOT PAID', 'Not Paid'
+        PAID = 'PAID', 'Paid'
```

Comment:
I think we only need PAID & NOT_PAID for statuses on this model.
NONE/ACCRUING will never be set because the penalty does not exist in these states
DUE = NOT_PAID